src/org/sosy_lab/java_smt/example/NQueens.java

```diff
@@ -0,0 +1,362 @@
+
+
+
+
+
+/*
+ *  JavaSMT is an API wrapper for a collection of SMT solvers.
+ *  This file is part of JavaSMT.
+ *
+ *  Copyright (C) 2007-2016  Dirk Beyer
+ *  All rights reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+
+
+
+
+
+
+/*
+*  The Output of this code
+
+
+
+
+* The Queen can be placed in these ways:
+0100
+0001
+1000
+0010
+
+* here '1' indicates position of queen that has been placed
+
+
+* */
+
+
+import com.google.common.base.Joiner;
```

Comment:
The order in a file should be as follows:
- legal notice
- imports
- JavaDoc for class description
- constant, fields
- constructor
- functions

Please apply a code-formatter like `ant format-source`. This might automatically solve several issues here.